api/audit/models.py

```diff
@@ -113,12 +109,9 @@ def add_project(self):
         when="environment_document_updated",
         is_now=True,
     )
-    def process_environment_update(self):
-        self.update_environments_updated_at()
-        self.send_environments_to_dynamodb()
-        self.send_environment_update_message()
-
     def update_environments_updated_at(self):
```

Comment:
This method feels like it could be named better since it's triggering another task to handle the environment update logic. It might also benefit from a docstring I guess?

Reply:
But it does update the `updated_at`

Reply:
I guess I can change it back to `process_environment_update`